hot/docker-host-stack-1.yaml

```diff
@@ -4,11 +4,15 @@ parameters:
   flavor:
     type: string
     description: Flavor to use
-    default: 2C-4GB-50GB
-  image:
+    default: b.2c2gb
+  image_name:
```

Comment:
`image` is actually fine, because to the best of my knowledge you can always reference the image by name (as long as it's unambiguous in the tenant) or UUID. I'd prefer if you kept `image`, unless you tested this with a UUID and it broke.

Reply:
Cool, I reverted to `image`.